bubblegum/program/Cargo.toml

```diff
@@ -15,13 +15,14 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-anchor-lang = { version = "0.25.0", features = [ "init-if-needed" ] }
+anchor-lang = { version = "0.25.0", features = ["init-if-needed"] }
 anchor-spl = { version = "0.25.0" }
-spl-token = { version = "3.3.0", features = [ "no-entrypoint" ] }
-spl-associated-token-account = { version = "1.0.5", features = [ "no-entrypoint" ] }
+spl-token = { version = "3.3.0", features = ["no-entrypoint"] }
+spl-associated-token-account = { version = "1.0.5", features = ["no-entrypoint"] }
 mpl-token-metadata = { version = "1.3.3", features = ["no-entrypoint"] }
-gummyroll = { git = "https://github.com/jarry-xiao/candyland.git", features = ["cpi"] }
+spl-compression = { git = "https://github.com/solana-labs/solana-program-library", features = ["cpi"] }
```

Comment:
I think it's a good practice for `git` dependencies to also refer to a specific commit using `rev = <hash>` in the specification, so we don't inadvertently end up consuming something unexpected just because a branch advances.